packages/playground/data-liberation/src/functions.php

```diff
@@ -191,3 +191,37 @@ function wp_visit_file_tree( $dir ) {
 		new SplFileInfo( $dir )
 	);
 }
+
+/**
+ * Import a WXR file. Used by the CLI.
+ *
+ * @param string $path The path to the WXR file.
+ * @return void
+ */
+function data_liberation_import( $path ): bool {
+	$importer = WP_Stream_Importer::create_for_wxr_file( $path );
+
+	if ( ! $importer ) {
+		return false;
+	}
+
+	$is_wp_cli = defined( 'WP_CLI' ) && WP_CLI;
```

Comment:
At this point, a dedicated WP_CLI command might make sense. It would only be a thin wrapper. The website and the unit tests would use the same underlying import library with their own dedicated logging facilities.

Reply:
Yes. I made this way so that a user that want to use only the plugin do not need to have WP-CLI all the times.